active-rfcs/0000-router-navigation-failures.md

```diff
@@ -165,9 +165,10 @@ router.afterEach((to, from, failure) => {
 
 # Alternatives
 
-- Differentiating `next(false)` from navigations that get overriden by more recent navigations by defining another Navigation Failure
+- Differentiating `next(false)` from navigations that get overridden by more recent navigations by defining another Navigation Failure
 
 # Adoption strategy
 
+- Expose `NavigationFailureType` in vue-router@3 so that Navigation Failures can be told apart from regular Errors. We could also expose a function `isNavigationFailure` to tell them apart.
```

Comment:
ahh, this will help too. is there a better way to check for navigation failures right now than `e && e.name === 'NavigationDuplicated'`?

Reply:
There is no documented way to check for duplicated navigations right now (it's private api)